specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation",
+                "type": "string"
+              },
+              "Location": {
+                "description": "Url to retrieve the final result after operation completes.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Search for phone numbers": {
+            "$ref": "./examples/SearchPhoneNumbersAsync.json"
+          }
+        }
+      }
+    },
+    "/availablePhoneNumbers/searchResults/{searchId}": {
+      "get": {
+        "tags": [
+          "Search"
+        ],
+        "summary": "Gets phone number search by search id.",
+        "operationId": "PhoneNumber_GetPhoneNumberSearch",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "searchId",
+            "description": "The search Id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SearchResponse"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get PhoneNumber Search Details": {
+            "$ref": "./examples/GetPhoneNumberSearchAsync.json"
+          }
+        }
+      }
+    },
+    "/availablePhoneNumbers/:purchase": {
+      "post": {
+        "tags": [
+          "Search"
+        ],
+        "summary": "Purchases the phone number search.",
+        "operationId": "PhoneNumber_PurchasePhoneNumberSearch",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberPurchaseRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
202 Response is missing shema:
"schema": {
  "$ref": "#/definitions/PhoneNumberSearchResult"
}